http/src/request/user/update_current_user.rs

```diff
@@ -44,6 +49,13 @@ impl<'a> UpdateCurrentUser<'a> {
         }
     }
 
+    /// Set the user's avatar.
+    ///
+    /// This must be a Data URI, in the form of `data:image/{type};base64,{data}` where `{type}` is
```

Comment:
do we want to use a semi colon here as an or? I know the semicolon isn't used much in average English conversations, so is this too obscure?

This is probably fine, but I just want to know if anyone else has thoughts on this.

Reply:
I've been copypasting this from other docs: webhook avatar and message attachments. In a separate PR I could change the formatting for all instances.

Reply:
Sounds good 👍